src/libraries/sendtohelixhelp.proj

```diff
@@ -29,6 +29,7 @@
     <_workItemTimeout Condition="'$(Scenario)' == '' and '$(_workItemTimeout)' == '' and ('$(TargetArchitecture)' == 'arm64' or '$(TargetArchitecture)' == 'arm')">00:45:00</_workItemTimeout>
     <_workItemTimeout Condition="'$(Scenario)' != '' and '$(_workItemTimeout)' == '' and ('$(TargetArchitecture)' == 'arm64' or '$(TargetArchitecture)' == 'arm')">01:00:00</_workItemTimeout>
     <_workItemTimeout Condition="'$(Scenario)' == 'BuildWasmApps' and '$(_workItemTimeout)' == ''">01:00:00</_workItemTimeout>
+    <_workItemTimeout Condition="'$(_workItemTimeout)' == '' and '$(TargetOS)' == 'Browser'">00:30:00</_workItemTimeout>
```

Comment:
I don't think it is ideal to bump all the `wasm` workitems to 30 mins just because one workitem is timing out. I think that is kind of dangerous to do, because if for some reason something causes a bunch of workitems to hang, instead of failing at 15 mins they will now fail at 30 mins. Potentially filling up the queue because the wait times will be pretty high.

I would suggest doing it for the System.Text.Json workitem, maybe when we build the helix workitem later in this file?

Reply:
Sure, whatever gets agreed upon.

Reply:
That works for me as well. Don't have a strong opinion here.

Reply:
Let's just apply the change to STJ and unblock CI for now.